packages/shader-lab/src/contentParser/ShaderContentParser.ts

```
@@ -57,6 +65,7 @@
   private static _symbolTable: SymbolTableStack<ISymbol, SymbolTable> = new SymbolTableStack();
 
   static reset() {
+    this._errors.length = 0;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)